### PR TITLE
permalink by build process for staging

### PR DESCRIPTION
**Pull request by chachasikes**
try setting an env var in github action workflow to be available to 11ty build

### CODE REVIEW

**Review by kkoryaka — APPROVED** [review]
Looks good

File: .github/workflows/eleventy_build.yml [hunk 1]
```diff
@@ -20,6 +20,8 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: ./docs
           publish_branch: deploy_production
+        env:
+          SITE_DOMAIN: "https://staging.drought.ca.gov"
```
Comment (aaronhans) [inline .github/workflows/eleventy_build.yml:24]: Don't have this subdomain setup yet but going to request it.

I don't know if this is the best way to solve this but we can setup more environments here https://github.com/cagov/drought.ca.gov/settings/environments, then define values that get pulled in to these build files as variables with the ${{ }} syntax

I made you admin on drought.ca.gov

**Review by aaronhans — APPROVED** [review]

File: .github/workflows/eleventy_build.yml (hunk 1 above)
Comment (chachasikes) [reply .github/workflows/eleventy_build.yml:24]: cool thank you

**Comment by chachasikes** [comment]
trying it

**Pull request merged**